src/components/Footer/Footer.js

```diff
@@ -28,8 +40,18 @@ const Footer = ({
   blockStatus,
   loadingVote,
   showScoreButton = false,
-  onPressScore
+  onPressScore,
+  isShowDM = false
 }) => {
+  const {sendMessageDM} = useDMMessage();
+  const [profile] = React.useContext(Context).profile;
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+  const {createSignChat} = useCreateChat();
+
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces several new state variables and hooks. However, there is no error handling or null checking for these new hooks and state variables. This could potentially lead to runtime errors if any of these hooks fail or return unexpected values. Consider adding appropriate error handling and null checks.


<!-- This is an auto-generated comment by OSS CodeRabbit -->